QUERIES.md

```diff
@@ -37,13 +37,19 @@ REVOKE <privilege> ON <db> FROM <user>
 -- revoke all privileges for a DB
 REVOKE ALL [PRIVILEGES] ON <db> FROM <user>
 
--- revoke all of user's privileges (all DBs and/or cluster admin)
+-- revoke all privileges including cluster admin
 REVOKE ALL [PRIVILEGES] FROM <user>
 
 -- delete a user
 DROP USER <name>
 ```
-where `<privilege> := READ | WRITE | All [PRIVILEGES]`.
+where `<privilege> := READ | WRITE | All [PRIVILEGES]`. 
```

Comment:
This line is correct but confusing. Might want to split up the statement into several lines for clarity. For example,

where `<privilege>` can be either:
- `READ`
- `WRITE`
- `ALL [PRIVILEGES]`